Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+/** <title>GSXib5KeyedUnarchiver.m</title>
+
+ <abstract>The XIB 5 keyed unarchiver</abstract>
+
+ Copyright (C) 2016,2017 Free Software Foundation, Inc.
+
+ Author:  Marcian Lytwyn <gnustep@advcsi.com>
+ Date: 12/28/16
+
+ This file is part of the GNUstep GUI Library.
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library; see the file COPYING.LIB.
+ If not, see <http://www.gnu.org/licenses/> or write to the
+ Free Software Foundation, 51 Franklin Street, Fifth Floor,
+ Boston, MA 02110-1301, USA.
+ */
+
+#import "GSXib5KeyedUnarchiver.h"
+#import "GNUstepGUI/GSNibLoading.h"
+#import "GNUstepGUI/GSXibLoading.h"
+#import "GNUstepGUI/GSXibElement.h"
+
+#import "AppKit/NSApplication.h"
+#import "AppKit/NSBox.h"
+#import "AppKit/NSBrowser.h"
+#import "AppKit/NSBrowserCell.h"
+#import "AppKit/NSButtonCell.h"
+#import "AppKit/NSCell.h"
+#import "AppKit/NSClipView.h"
+#import "AppKit/NSFormCell.h"
+#import "AppKit/NSImage.h"
+#import "AppKit/NSMatrix.h"
+#import "AppKit/NSMenu.h"
+#import "AppKit/NSMenuItem.h"
+#import "AppKit/NSNib.h"
+#import "AppKit/NSParagraphStyle.h"
+#import "AppKit/NSPopUpButton.h"
+#import "AppKit/NSPopUpButtonCell.h"
+#import "AppKit/NSScroller.h"
+#import "AppKit/NSScrollView.h"
+#import "AppKit/NSSliderCell.h"
+#import "AppKit/NSSplitView.h"
+#import "AppKit/NSTableColumn.h"
+#import "AppKit/NSTableHeaderView.h"
+#import "AppKit/NSTableView.h"
+#import "AppKit/NSTabView.h"
+#import "AppKit/NSToolbarItem.h"
+#import "AppKit/NSView.h"
+#import "GSCodingFlags.h"
+
+//#define DEBUG_XIB5
```

Comment:
Hm, I don't really like commented-out code even if it's there for documentation.

Also, `#ifdef` IMHO should often be an `#if` based on the value of the macro, not an `#ifdef`. Which would make it possible to `#define DEBUG_XIB5 0` here (possibly with an `#ifndef DEBUG_XIB5` to allow to pass the `-D` flag to the compiler to change this).